### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by knysh**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+            <name>Nikita Kuznetsov</name>
+        </developer>
+        <developer>
+            <id>mialeska</id>
+            <name>Alaksiej Mialeška</name>
+        </developer>
+        <developer>
+            <id>sunigos</id>
+            <name>Igor Sontsa</name>
+        </developer>
```
Comment (mialeska) [inline pom.xml]: Please add yourself :)

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- task: Maven@3
+  inputs:
+    mavenPomFile: 'pom.xml'
+    mavenOptions: '-Xmx3072m'
+    javaHomeOption: 'JDKVersion'
+    jdkVersionOption: '1.8'
+    jdkArchitectureOption: 'x64'
+    publishJUnitResults: true
+    testResultsFiles: '**/surefire-reports/TEST-*.xml'
+    goals: 'clean test -DdriverSettings.chrome.webDriverVersion=75.0.3770.140 -Dprofile=local'
```
Comment (mialeska) [inline azure-pipelines.yml]: probably webDriverVersion is not actual for windows-latest image

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2019 Aquality Automation
```
Comment (mialeska) [inline LICENSE]: 2020

File: pom.xml (hunk 1 above)
Comment (knysh) [reply pom.xml]: okay :)

File: azure-pipelines.yml (hunk 2 above)
Comment (knysh) [reply azure-pipelines.yml]: agree

File: LICENSE (hunk 3 above)
Comment (knysh) [reply LICENSE]: fixed

**Pull request merged**